src/model/censusdata/censusdata.js

```diff
@@ -1,17 +1,18 @@
 import { writable, get } from "svelte/store";
 import { mapBBoxCodes, toggleable } from "./stores";
-import { addNewGeoDataToCache } from "../utils";
+import { addNewGeoDataToCache, catAndTotalCodesRequested } from "../utils";
 import config from "../../config";
+import { censusMetadata } from "../metadata/metadata";
 
 export let selectedGeographyData = writable(new Map());
 export let dataByGeography = writable(new Map());
 export let englandAndWalesData = writable(new Map());
 export let newDataByGeography = toggleable(false);
+export let cachedMapCategories = writable(new Set());
 
 export let censusTableStructureIsLoaded = writable(false);
```

Comment:
this is a metadata thing right? Does this belong here, in that case, or in the metadata service?

Reply:
no, it's a list of categories that we have made the kind of fetch requests for that would allow you to populate the map with data.  at this point that's just fetchSelectedDataForGeoType, but will extend to boundingBox requests etc

Reply:
not sure I get it, might need to go through this on a call.